### PR TITLE
Passing provided attributes to SVG tag

### PR DESCRIPTION
**Pull request by EtienneBerube**
Currently, there is no way to pass HTML attributes to the SVG tag. However, there are some attributes and styles that need to be on the SVG tag directly. 

For example, `height` and `width` have to be specified on the tag itself, otherwise it will result in the SVG taking 300px x 150px.

This PR would ensure that we can pass the necessary styles to the underlying SVG tag.

### CODE REVIEW

File: lib/action_view/use/helpers.rb [hunk 1]
```diff
@@ -3,11 +3,11 @@
 module ActionView
   module Use
     module Helpers
-      def use_symbol(name)
+      def use_symbol(name, **opts)
```
Comment (skipkayhil) [inline lib/action_view/use/helpers.rb]:
```suggestion
      def use_symbol(name, escape: true, **opts)
```

File: lib/action_view/use/helpers.rb [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
         used_symbol_set << name
 
         <<~HTML.html_safe
-          <svg xmlns="http://www.w3.org/2000/svg">
+          <svg xmlns="http://www.w3.org/2000/svg" #{attributes_string(opts)}>
```
Comment (skipkayhil) [inline lib/action_view/use/helpers.rb]:
```suggestion
          <svg xmlns="http://www.w3.org/2000/svg" #{tag_builder.tag_options(opts, escape)}>
```

File: lib/action_view/use/helpers.rb [hunk 3]
```diff
@@ -30,6 +30,10 @@ def all_used_symbols
         def used_symbol_set
           @used_symbol_set ||= Set.new
         end
+
+        def attributes_string(attrs)
+          attrs.map { |k, v| "#{k.to_s.dasherize}=\"#{v}\"" if v}.join(" ")
+        end
```
Comment (skipkayhil) [inline lib/action_view/use/helpers.rb]:
```suggestion
```

**Review by skipkayhil — COMMENTED** [review]
Action View actually has this method built in, so we can use it here

Also if you rebase on main, I added a real CI to run against

File: lib/action_view/use/helpers.rb (hunk 3 above)
Comment (skipkayhil) [reply lib/action_view/use/helpers.rb]: I think this method still needs to be removed 🙂 

(also can you squash the commits down to one)

Comment (EtienneBerube) [reply lib/action_view/use/helpers.rb]: Good catch! 

Will do!

Comment (EtienneBerube) [reply lib/action_view/use/helpers.rb]: Done!

**Pull request merged**